.github/workflows/testAndPublish.yml

```diff
@@ -245,6 +245,46 @@ jobs:
         annotate_only: true
         report_paths: testOutput/system/systemTests.xml
 
+  uploadSymbols:
+    name: Upload symbols
+    runs-on: windows-latest
+    needs: buildNVDA
+    steps:
+    - name: Cancel if feature disabled
```

Comment:
Can't we just avoid the job to run?

Reply:
no - env variables can't be checked at that level